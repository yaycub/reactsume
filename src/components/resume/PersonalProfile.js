import React from 'react';
import PropTypes from 'prop-types';

const PersonalProfile = ({ personalProfile }) => {
  return (
    <p>
      {personalProfile}
    </p>
  );
};

PersonalProfile.propTypes = {
  personalProfile: PropTypes.string.isRequired
};

export default PersonalProfile;
