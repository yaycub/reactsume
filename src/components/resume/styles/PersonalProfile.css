.Profile{
  display: flex;
  flex-direction: column;

  h3{
    font-size: 15px;
    font-weight: bold;
    margin-bottom: 10px;
  }

  p{
    font-size: 13px;
    margin-bottom: 10px;
  }
}