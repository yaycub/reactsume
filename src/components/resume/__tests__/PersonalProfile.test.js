import React from 'react';
import { shallow } from 'enzyme';
import PersonalProfile from '../PersonalProfile';

describe('PersonalProfile Component', () => {
  it('can render a PersonalProfile', () => {
    const wrapper = shallow(<PersonalProfile personalProfile={'This is my profile'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
