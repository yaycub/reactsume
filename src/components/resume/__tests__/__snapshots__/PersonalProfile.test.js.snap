// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalProfile Component can render a PersonalProfile 1`] = `
<div
  className="Profile"
>
  <h3>
    Summary
  </h3>
  <p>
    This is my profile
  </p>
</div>
`;
